### PR TITLE
docs: update releasing and support policy

### PR DESCRIPTION
**Pull request by kpanot**
## Proposed change

Update releasing and support policy as discussed within the responsible team.

## Related issues

*- No issue associated -*

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -20,9 +20,23 @@ Two major versions are released per year:
...
+The "support" of a released version includes the following aspects:
 
-A major version will be supported as long as it relies on a [non deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule).
+- Handle bug fixing when requested via the [Github Issue page](https://github.com/AmadeusITGroup/otter/issues).
+- Review (and handle merging) of bugfix pull requests contribution.
+- Cascade bugfixes when contributed into inferior released version.
+- Handle bugfix cherry-picking on specific version demand.
+
+> [!WARNING]
+> Only **bugfix** are accepted on supported versions. A new feature are allowed exclusively on the `main` branch (latest *alpha* version).
```
Comment (matthieu-crouzet) [inline SECURITY.md:31]:
```suggestion
> Only **bugfix** are accepted on supported versions. A new feature are allowed exclusively on the `main` branch (latest *prerelease* version) and the `next` branch.
```

File: SECURITY.md [hunk 2]
```diff
@@ -20,9 +20,23 @@ Two major versions are released per year:
...
+The "support" of a released version includes the following aspects:
 
-A major version will be supported as long as it relies on a [non deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule).
+- Handle bug fixing when requested via the [Github Issue page](https://github.com/AmadeusITGroup/otter/issues).
+- Review (and handle merging) of bugfix pull requests contribution.
+- Cascade bugfixes when contributed into inferior released version.
+- Handle bugfix cherry-picking on specific version demand.
+
+> [!WARNING]
+> Only **bugfix** are accepted on supported versions. A new feature are allowed exclusively on the `main` (latest *prerelease* version) and the `next` branches.
```
Comment (cpaulve-1A) [inline SECURITY.md:31]:
```suggestion
> Only **bugfix** are accepted on supported versions. New features are allowed exclusively on the `main` (latest *prerelease* version) and the `next` branches. Note that breaking changes should always target a `next` branch.
```

File: SECURITY.md [hunk 3]
```diff
@@ -20,9 +20,23 @@ Two major versions are released per year:
...
+
+The team will support the latest minor version of major versions **N (current)**, **N-1** and **N-2**.\
+All the minors versions of the major **N (current)** are also supported as long as a new major version is not created.
+
+When a new major version **N** is released, all the minors version of the major **N-1**, except the latest one, will be considered as non-supported.\
+The major becoming **N-3** will be also flagged as non-supported.
```
Comment (cpaulve-1A) [inline SECURITY.md:37]:
```suggestion
The major becoming **N-3** will also be flagged as non-supported.
```

File: SECURITY.md [hunk 4]
```diff
@@ -20,9 +20,23 @@ Two major versions are released per year:
...
+
+A major version will be supported as long as it relies on a [non deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule) and included in the previously listed major versions.
```
Comment (cpaulve-1A) [inline SECURITY.md:39]:
```suggestion
A major version will be supported as long as it relies on a [non-deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule) and is included in the previously listed major versions.
```

File: SECURITY.md [hunk 5]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
 
 ## Release version support
 
-The team will provide support on the latest minor version of major versions **N (current)**, **N-1** and **N-2**.
+The "support" of a released version includes the following aspects:
 
-A major version will be supported as long as it relies on a [non deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule).
+- Handle bug fixing when requested via the [Github Issue page](https://github.com/AmadeusITGroup/otter/issues).
```
Comment (sdo-1A) [inline SECURITY.md:25]:
```suggestion
- Handling bug fixes when requested via the [Github Issue page](https://github.com/AmadeusITGroup/otter/issues).
```

File: SECURITY.md [hunk 6]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+- Review (and handle merging) of bugfix pull requests contribution.
```
Comment (sdo-1A) [inline SECURITY.md:26]:
```suggestion
- Reviewing contributions to bugfix pull requests (and handling their merging).
```

File: SECURITY.md [hunk 7]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+- Cascade bugfixes when contributed into inferior released version.
```
Comment (sdo-1A) [inline SECURITY.md:27]:
```suggestion
- Cascading bugfixes when they are implemented in an inferior released version.
```

File: SECURITY.md [hunk 8]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+- Handle bugfix cherry-picking on specific version demand.
```
Comment (sdo-1A) [inline SECURITY.md:28]:
```suggestion
- Handling bugfix cherry-picking on a specific version demand.
```

File: SECURITY.md [hunk 9]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+
+> [!WARNING]
+> Only **bugfix** are accepted on supported versions. New features are allowed exclusively on the `main` (latest *prerelease* version) and the `next` branches.
```
Comment (sdo-1A) [inline SECURITY.md:31]:
```suggestion
> Only **bugfixes** are accepted on supported versions. New features are allowed exclusively on the `main` (latest *prerelease* version) and the `next` branches.
```

File: SECURITY.md [hunk 10]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+> Note that **breaking changes** should always target a `next` branch.
+
+The team will support the latest minor version of major versions **N (current)**, **N-1** and **N-2**.\
+All the minors versions of the major **N (current)** are also supported as long as a new major version is not created.
```
Comment (sdo-1A) [inline SECURITY.md:35]:
```suggestion
All the minor versions of the major **N (current)** are also supported until a new major version is created.
```

File: SECURITY.md [hunk 11]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+The major becoming **N-3** will also be flagged as unsupported.
```
Comment (sdo-1A) [inline SECURITY.md:36]:
```suggestion
The major version becoming **N-3** will also be flagged as unsupported.
```

File: SECURITY.md [hunk 12]
```diff
@@ -20,9 +20,22 @@ Two major versions are released per year:
...
+The "support" of a released version includes the following aspects:
 
-A major version will be supported as long as it relies on a [non deprecated version of Angular](https://angular.io/guide/releases#support-policy-and-schedule).
+- Handling bug fixes when requested via the [Github Issue page](https://github.com/AmadeusITGroup/otter/issues).
+- Reviewing contributions to bugfix pull requests (and handling their merging).
+- Cascading bugfixes when they are implemented in an inferior released version.
+- Handling bugfix cherry-picking on a specific version demand.
+
+> [!WARNING]
+> Only **bugfixes** are accepted on supported versions. New features are allowed exclusively on the `main` (latest *prerelease* version) and the `next` branches.
```
Comment (mrednic-1A) [inline SECURITY.md:31]: **Bugfixes** are only accepted on supported versions ?

**Review by mrednic-1A — APPROVED** [review]

**Review by sdo-1A — APPROVED** [review]

**Pull request merged**